src/activitylogger/content_script/index.js

```diff
@@ -1,10 +1,30 @@
 import { getMetadata, metadataRules } from 'page-metadata-parser'
+import { Readability } from 'readability'
 
 // Get the basic info and text content (for search index) of this page.
 function gatherPageInfo() {
     // Extract info from all sorts of meta tags (og, twitter, etc.)
     const visitedUrl = window.location.href
-    const text = document.body.innerText
+
+    const location = document.location
+    const uri = {
+        spec: location.href,
+        host: location.host,
+        prePath: location.protocol + "//" + location.host,
+        scheme: location.protocol.substr(0, location.protocol.indexOf(":")),
+        pathBase: location.protocol + "//" + location.host + location.pathname.substr(0, location.pathname.lastIndexOf("/") + 1)
+    };
+    const documentClone = document.cloneNode(true)
+    const article = new Readability(uri, documentClone).parse()
+    let text;
+    if (article) {
+        const articleDom = document.createElement('main')
+        articleDom.innerHTML = article.content
+        text = articleDom.innerText
```

Comment:
Argh, because it isn't mentioned in the [readme](https://github.com/mozilla/readability/blob/cec7d90a14b977ea80df12864c311b160f74dc34/README.md#usage). So go for `article.textContent`.